.taskcluster.yml

```diff
@@ -85,8 +87,14 @@ tasks:
               image_tag: '3.6'
             - short_name: py37
               image_tag: '3.7'
+            - short_name: py38
+              image_tag: '3.8'
+            - short_name: py39
+              image_tag: '3.9'
+            - short_name: py310
+              image_tag: '3.10'
             - short_name: docs-localbuild
-              image_tag: '3.7'
+              image_tag: '3.10'
```

Comment:
We might want to make the python versions in docs-localbuild and readthedocs match?